rpc/lib/client/ws_client.go

```diff
@@ -111,8 +111,8 @@ func (wsc *WSClient) receiveEventsRoutine() {
 				wsc.ErrorsCh <- err
 				continue
 			}
-			if response.Error != "" {
-				wsc.ErrorsCh <- errors.Errorf(response.Error)
+			if response.Error != nil && response.Error.Message != "" {
```

Comment:
So what happens to all the other, response.Error's that have empty messages?
Will they all now just fall through and be treated as success?

Perhaps we can tweak this a little e.g
```go
if response.Error != nil {
   switch response.Error.Message {
   case "":
       wsc.ErrorsCh <- errors.Errorf(response.Error)
   default:
       wsc.ErrorsCh <- errors.Errorf(response.Error.Message)
   }
}
```
that way we can catch all errors.
PS: I obviously don't have much context about the code base since I noticed the change from a string to nullable type :)